dev/Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM ubuntu:18.04
+FROM tensorflow/tensorflow:custom-op
 
 RUN apt-get update && \
     apt-get install -y \
-    build-essential \
     curl \
     nano \
     unzip \
     ffmpeg
 
-ARG BAZEL_VERSION=0.20.0
+ARG BAZEL_VERSION=0.24.1
```

Comment:
@yongtang I think you said 0.24.1 is proffered to build TF, so updating that here